tests/test_recipe_synth_additional.py

```
@@ -0,0 +1,159 @@
+        cuisine_type="intl",
+        base_ingredients=["a"],
+        cooking_methods=["b"],
+        typical_prep_time=1,
+        typical_cook_time=2,
+        difficulty="easy",
+        instruction_template="Do things.",
+        nutrition_profile={"calories": 1.0},
+    )
+    d = t.to_dict()
+    assert isinstance(d, dict) and d["template_id"] == "x"
+    # ensure it’s a shallow copy, not a live view
```

Ruff (0.14.1)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)